src/mongoose_graphql/admin/mongoose_graphql_admin_stanza_mutation.erl

```diff
@@ -0,0 +1,66 @@
+-module(mongoose_graphql_admin_stanza_mutation).
+
+-export([execute/4]).
+
+-ignore_xref([execute/4]).
+
+-include("../mongoose_graphql_types.hrl").
+-include("mongoose_logger.hrl").
+-include("jlib.hrl").
+
+execute(_Ctx, _Obj, <<"sendMessage">>, Opts) ->
```

Comment:
What's happened when I send a message from a nonexisting user to a nonexisting user?